pages/common/ansible.md

```diff
@@ -0,0 +1,27 @@
+# ansible
+
+> Manage groups of computers remotely over SSH
+
+- Execute a command on a group of hosts:
```

Comment:
Doing useful things in Ansible requires invoking modules ([full list](http://docs.ansible.com/ansible/list_of_all_modules.html)). Thus, we specify:

- **-m (module name):** The Ansible module to run
- **-a (module args):** Arguments for that module

The most straightforward module to use would be the [command module](docs.ansible.com/ansible/command_module.html), which lets users run ad-hoc commands on remote machines. It's usually shown in beginner tutorials. e.g. https://youtu.be/MfoAb50Br94?t=189